docs/djdknativedigest.md

```diff
@@ -38,7 +38,9 @@ This option enables or disables OpenSSL native cryptographic support for the Dig
 
 <!--OpenSSL support is enabled by default for the Digest algorithm. If you want to turn off this algorithm only, set this option to `false`.-->
 
- To turn off all the algorithms, see the [-Djdk.nativeCrypto](djdknativecrypto.md) system property command line option.
+ To turn off all the algorithms, see the [-Djdk.nativeCrypto](djdknativecrypto.md) system property command-line option.
+
+ :fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** MD5 digest algorithm is not supported for OpenSSL on AIX&reg;. This option is ignored on AIX and the Java implementation is always used.
```

Comment:
The MD5 ...

As per the above, the option also controls SHA digests so the following is not correct.
"This option is ignored on AIX and the Java implementation is always used."

Reply:
That's right. The option works fine. It just doesn't control `MD5` which is always done through Java on AIX.